controller/Makefile

```diff
@@ -4,6 +4,8 @@ include ../includes.mk
 
 all: build run
 
+SHELL_SCRIPTS = build.sh $(wildcard bin/*) $(wildcard migrations/data/*.sh)
```

Comment:
That works! I didn't know if `.` would demonstrate the same behaviour as $PWD. I thought Makefiles were smarter than that. Good catch!